content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+---
+author: ricardobaeta
+date: 2018-07-09
+title: "Cheers to small beginnings!"
+image: /post/cheers-to-small-beginnings/{d}ali-banner.png
+description: "A closer look at source{d}'s rebranding."
+categories: ["design", "branding"]
+draft: true
+---
+
+## A closer look at source{d}’s rebranding
+
```

Comment:
Add a quick intro. What is this blog post about?

Reply:
See the last commit :) Thanks!